src/web/templates/user/screen/results.html

```diff
@@ -18,13 +18,13 @@ <h2 class="set-title text-center">{{ screen.name }}</h2>
                 <tbody>
                 {% for result in results %}
                     <tr class="result-row">
-                        <td scope="row" class="time bg-transparent text-center">{{ result.timestamp_str }}</td>
-                        {% with tournament=user_event.tournaments_by_id[result.tournament_id] %}
+                        <td scope="row" class="time bg-transparent text-center">{{ result.last_result_update_str }}</td>
+                        {% with tournament=user_event.tournaments_by_id[result.tournament.id] %}
                         {% with players_by_id=tournament.players_by_id %}
-                        {% with white_player=players_by_id[result.white_player_id] %}
-                        {% with black_player=players_by_id[result.black_player_id] %}
+                        {% with white_player=players_by_id[result.white_player.id] %}
+                        {% with black_player=players_by_id[result.black_player.id] %}
                         {% if print_tournament %}<td scope="row" class="tournament bg-transparent">{{ tournament.uniq_id }}</td>{% endif %}
-                        <td scope="row" class="board-number bg-transparent text-center">{{ result.round }}.{{ result.board_id }}</td>
+                        <td scope="row" class="board-number bg-transparent text-center">{{ result.round }}.{{ result.id }}</td>
```

Comment:
```suggestion
                        <td scope="row" class="board-number bg-transparent text-center">{{ result.round }}.{{ result.index }}</td>
```
`Board.id` is legacy, as it does not refer to the Database ID as it would be expected